addons/project/static/src/views/project_task_kanban/project_task_kanban_record_quick_create.js

```diff
@@ -0,0 +1,10 @@
+/** @odoo-module */
+
+import { KanbanRecordQuickCreate } from "@web/views/kanban/kanban_record_quick_create";
+
+export class ProjectTaskKanbanRecordQuickCreate extends KanbanRecordQuickCreate {
+    async getQuickCreateProps(props) {
+        await super.getQuickCreateProps(props);
+        delete this.quickCreateProps.context.default_personal_stage_type_ids
```

Comment:
```suggestion
        const res = await super.getQuickCreateProps(props);
        delete this.quickCreateProps.context.default_personal_stage_type_ids;
        return res;
```
Shouldn't we return something? :thinking: